core/scripts/deploy-daemon.sh

```diff
@@ -21,8 +21,11 @@ then
   echo "${green}Amber compiled.${reset}"
   echo
 
+  echo "${green}Compiling WorkflowCompilingService...${reset}"
```

Comment:
Make sure -s skips compilation of workflowcompling service as well

Reply:
Sure. It will skip